editor.planx.uk/src/@planx/components/MapAndLabel/test/mocks/GenericValues.ts

```diff
@@ -1,15 +1,11 @@
 export type TreeData = {
   species: string;
   work: string;
```

Comment:
now that proposed work is `required: false` in the schema, we'll want the type here to be `work?: string;` - this should fix a couple of the test errors (which have still been naivly assuming everything `required` inconsistent with recent `List` changes !)